examples/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "tinyman-js-sdk-examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start:dev": "nodemon"
```

Comment:
Maybe we can add start script too

```suggestion
  "scripts": {
    "start": "ts-node ./src/index.ts",
    "start:dev": "nodemon"
```

Reply:
On second thought, we won't need nodemon anymore, it was more useful for developing examples. So I removed it, thanks!